app/src/test/java/co/ostorlab/insecure_app/BugRuleCallerTest.java

```diff
@@ -234,4 +235,12 @@ public void ruleCaller_BiometricFingerprintPromptVulnerability_NoExceptionThrown
 
         Assert.assertEquals(caller.getRules().size(), 1);
     }
+
+    @Test
+    public void ruleCaller_PackageContext_NoExceptionThrown() throws Exception{
```

Comment:
Please update the markdown to reference this vulnerability and explain it.

Reply:
Ping on this comment.

Reply:
https://github.com/Ostorlab/KB/pull/28

Reply:
I meant the README of this repo: https://github.com/Ostorlab/ostorlab_insecure_android_app/blob/master/README.markdown